### DISCUSSION TITLE
In Issues: so easy to slip from Created, to Assigned, panels

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#2812] [Mobile]
Here in the Android app the slightest accidental direction mishap with the thumb, will cause the screen to slide from the created column into the assigned column. It happens all the time. 
![16497-1](https://user-images.githubusercontent.com/1230959/100397473-d3133600-3084-11eb-956d-406146b831ca.jpg)

**Comment by candyho** [comment]
Hey thanks for your feedback! We are using the default Android component for tabs and we haven't really tweaked the sensitivity of it. 🤔 Which Android OS and device are you using?

**Reply by jidanni** [reply]
ASUS Zenfone 3, Android 8.0, with fonts and screen size etc. cranked up to the max "for older users."
